matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/PatientInMatch.java

```diff
@@ -132,11 +133,6 @@
      */
     Patient getPatient();
 
-    /**
-     * @return patient href.
-     */
-    String getHref();
```

Comment:
As discussed, for existing MME matches there may be data in the `href` column which is  not present anywhere else, so it is important not to lose it and display it as we used to display before this change

Reply:
we can either write a migrator or (less work) just use the data from the column if it is available